build/server-image/kube-apiserver/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file create the kube-apiserver image.
+ARG BASEIMAGE
+# we use the hosts platform to apply the capabilities to avoid the need
+# to setup qemu for the builder.
+FROM --platform=linux/$BUILDARCH k8s.gcr.io/build-image/setcap:buster-v1.4.0
```

Comment:
for anyone else following along: see https://github.com/kubernetes/kubernetes/pull/96134#discussion_r580704333

Reply:
We should perhaps parameterize this image https://github.com/kubernetes/kubernetes/issues/99238#issuecomment-782789925

Reply:
Opened https://github.com/kubernetes/kubernetes/pull/99379 to address it.